views/next-steps.erb

```diff
@@ -0,0 +1,117 @@
+<% content_for :head do %>
+    <title>Start using GOV.UK PaaS - <%= settings.title %></title>
+<% end %>
+
+<% content_for :breadcrumb do %>
+    <%== erb :'partials/_breadcrumb', :locals => {name: 'Get started', href: '/get-started'} %>
+    <%== erb :'partials/_breadcrumb', :locals => {name: 'Start using GOV.UK PaaS', href: '/next-steps', active: true} %>
+<% end %>
+
+<div class="container">
+  <div class="grid-row">
+    <% if params.key?('success') %>
+        <div class="column-full">
+          <div class="govuk-box-highlight-green account-created">
+            <p>Your account has been activated</p>
+          </div>
+        </div>
+    <% end %>
+
+    <div class="column-two-thirds">
+      <h1 class="heading-large">
+        Start using GOV.UK PaaS
+      </h1>
+
+      <p class="lede">
+        A lot of the tasks you’ll need to do to operate GOV.UK PaaS will be done from your computers
+        command line interface.
+      <p>
+    </div>
+  </div>
+
+  <div class="grid-row download-section">
+    <div class="column-two-thirds">
+      <h2 class="heading-small">
+        Download the Cloud Foundry command line tool
+      </h2>
+
+      <p>
+        You’ll need to do this to operate GOV.UK PaaS from your terminal. You can download the cf command line tool
+        using one of the installers shown. You can also visit the github page for
+        <a href="https://github.com/cloudfoundry/cli#downloads">other download options</a> such as using package
+        managers.
+      </p>
+    </div>
+
+    <div class="column-one-third">
+      <p class="heading-small">Installation options: </p>
+
+      <ul class="list-bullet">
+        <li><a href="https://packages.cloudfoundry.org/stable?release=macosx64&source=github">Mac OS X 64 bit</a></li>
+        <li><a href="https://packages.cloudfoundry.org/stable?release=windows64&source=github">Windows 64 bit</a></li>
+        <li><a href="https://packages.cloudfoundry.org/stable?release=debian64&source=github">Debian 64 bit</a></li>
+        <li><a href="https://packages.cloudfoundry.org/stable?release=redhat64&source=github">RedHat 64 bit</a></li>
```

Comment:
Strictly, these should be `&amp;`

Reply:
Changed. It isn't an issue if we're using HTML5 which we are by defining the `<!doctype html>`.

It will however try to parse it as an entity in lower HTML version and does no harm in our case to have `&amp;`. Going with this example there are many more errors/warnings when you run a validator as many of HTML5 compliant tags will fail the HTML4 check.